frontend/src/app/hierarchy-viewer/importExportHierarchy.dialog.html

```diff
@@ -0,0 +1,39 @@
+<mat-tab-group (selectedIndexChange)="myTabSelectedIndexChange($event)">
+    [selectedIndex]="selectedTab.value">
+    <mat-tab label="Import">
+        <mat-dialog-content class="tab-container">
+            <input type="file" class="input-hidden" (change)="onFileSelected($event)" #fileUpload>
+            <p class="file-name">{{fileName || "No file uploaded yet."}}</p>
+            <button mat-raised-button color="accent" class="upload-btn" (click)="fileUpload.click()">
+                <mat-icon>attach_file</mat-icon>
+            </button>
+        </mat-dialog-content>
+    </mat-tab>
+    <mat-tab label="Export">
+        <mat-dialog-content class="tab-container">
```

Comment:
This dialog content isn't really necessary. Maybe you could just show the name of the hierarchy that's being exported? Or you could get rid of it and just keep the button. Also, I think the export tab should be hidden if no hierarchy is selected.